mlflow/entities/span.py

```diff
@@ -28,32 +29,28 @@ class SpanType:
 
 @dataclass
 class Span(_MlflowObject):
```

Comment:
note: These entities will be removed anyway in follow-up, so feel free to skip if the JSON schema itself looks good.